packages/server/src/ratelimit.ts

```diff
@@ -1,10 +1,15 @@
+import { tooManyRequests } from '@medplum/core';
+import { Request } from 'express';
 import rateLimit, { MemoryStore, RateLimitRequestHandler } from 'express-rate-limit';
-import { OperationOutcomeError, tooManyRequests } from '@medplum/core';
+import { AuthenticatedRequestContext, getRequestContext } from './context';
 
 /*
  * MemoryStore must be shutdown to cleanly stop the server.
  */
 
+const DEFAULT_RATE_LIMIT_PER_15_MINUTES = 10000;
```

Comment:
Could you add a brief note [here](https://www.medplum.com/docs/fhir-datastore) with the new default rate limit>